repositories.txt

```diff
@@ -1173,7 +1173,7 @@ https://github.com/fabiopjve/ULWOS2
 https://github.com/fabiuz7/Dimmable-Light-Arduino
 https://github.com/fabiuz7/esp-logger-lib
 https://github.com/fabiuz7/melody-player-arduino
-https://github.com/fabiuz7/rtc-memory-esp8266
+https://github.com/fabianoriccardi/RTCMemory-ESP8266
```

Comment:
Hi @fabianoriccardi I was about to perform the manual update operations in our crawler, but I saw that the repo now is called
https://github.com/fabianoriccardi/rtcmemory
So I guess it's better to update the PR to use directly the final URL, before I start the manual operations, Can you please update this PR?

In addition, I think it's better also to update your `library.properties` here 
https://github.com/fabianoriccardi/rtcmemory/blob/cc8ddbef5a41f1313d05d9d68768f8cac7276f82/library.properties#L7
to the final github repo link.

Thanks!

Reply:
>use directly the final URL, before I start the manual operations

I agree that is the best thing and have submitted a follow-up PR for that here: https://github.com/arduino/library-registry/pull/293

I also updated the outstanding PR for the related changes to registry.txt: https://github.com/arduino/library-registry/pull/130